src/view/Cinema.js

```diff
@@ -0,0 +1,63 @@
+import starFilled from '../assets//../assets/star_filled.png';
+import { IMAGE_BASE_URL } from '../constants/api';
+import { MovieCard } from '../components/MovieCard';
+
+export class Cinema {
+    constructor() {
+        this.isConnect = false;
+        this.page = 1;
+        this.movieListElement = document.querySelector('.item-list');
+    }
```

Comment:
멤버변수는 클래스 필드에 미리 정의해주세요!

```suggestion
export class Cinema {
    isConnect;
    page;
    movieListElement;
    
    constructor() {
        this.isConnect = false;
        this.page = 1;
        this.movieListElement = document.querySelector('.item-list');
    }
```

Reply:
넵!! 클래스 필드에 미리 정의하겠습니다.